vspace/src/main/java/edu/asu/diging/vspace/core/exception/LanguageListConfigurationNotFound.java

```diff
@@ -0,0 +1,27 @@
+package edu.asu.diging.vspace.core.exception;
+
+public class LanguageListConfigurationNotFound  extends Exception {
```

Comment:
this should be a runtime exception and exception class names should end in `Exception`